docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -609,106 +612,347 @@ simulants are dying at the expected rates.
-infant. We could add functionality such that children move with their
-mothers from birth up to some fixed age (or something similar), but
-for now we will have this limitation that our migration model does not
-take family structure into account.
-
-These notes on ACS data sources on migration could be useful for the
-more complex rates in the future.  Based on age, sex, race/ethnicity,
-and geography, we can calculate the probability of moving from ACS, as
-the weighted average of MIGPUM.isnull(); could also determine if they
-moved within the PUMAs represented in the sim or from outside those
-PUMAs.
-For now, we only model migration within the sim catchment area (this component)
-and to/from other countries (next two components).
-When the simulation only includes part of the US, there is no domestic
-migration into or out of this region.
-
-Note that each housing unit in C should be associated with a unique
-mailing address, as described in Section (8).
-
-We might also want to think about the change
-in relationship type when people move, and also change surnames
-sometimes.
-
-We might also put a "demographic" model on the housing units in (C);
-according to `ACS: America's Data At Risk
-(p. 21) <https://censusproject.files.wordpress.com/2022/03/census_white-paper_final_march_2022.pdf>`_,
-"Between 2000 and 2019, the number of housing units increased by 23.8
-million or almost 21%."
-
-But to summarize, for our initial implementation, here are the
-simplifying assumptions that we have included:
-
-#. each household will have one address
-
-#. when a household moves, we will create a new address for them. no
-   one will move back into that old address.
+returns might not have a protected identification key (PIK).
 
-#. each time an individual moves, they move into an existing household
-   / household id. this household is chosen at random out of all
-   households excluding their current one
+Data sources
+^^^^^^^^^^^^
 
-#. each time an individual moves into an existing household, they gain
-   the relationship to head of household "Other nonrelative"
+All data comes from ACS PUMS.
+We use the standard columns about demographics, household structure, etc.
+We also use some that are specifically relevant to moving:
 
-#. the head of household cannot move to a new household
+* What PUMA the person lives in now (:code:`ST` and :code:`PUMA`).
+* Whether they moved domestically in the last 12 months (:code:`MIG` and :code:`MIGSP`).
+* If they moved, what "migration PUMA" (MIGPUMA) they lived in 12 months ago (:code:`MIGSP` and :code:`MIGPUMA`).
 
-#. Group Quarters address and zip code do not change
+MIGPUMAs are geographic entities created for this purpose.
+They are similar to PUMAs and many are exactly identical to a PUMA.
+However, some individual PUMAs had too few people moving from them, so they were grouped together
+with neighboring PUMAs into a single MIGPUMA for disclosure avoidance reasons.
```

Comment:
out of curiosity, when this is the case, what does the PUMA variable contain?

Reply:
For the input files to the simulation, I have handled this wrinkle for you. Both the source and destination columns are regular PUMAs.

(Because we don't have granularity below MIGPUMA for the source, that means I've duplicated rows for each source PUMA that lies within the MIGPUMA.)